.github/workflows/dependabot_pr.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       - name: Update the changelog
         uses: dangoslen/dependabot-changelog-helper@v3
         with:
-          version: 'Unreleased 3.0'
+          version: 'Unreleased 2.x'
```

Comment:
Not sure how this workflow works. 
I would love to understand it.

Reply:
This workflow runs on top of the dependabot workflow which updates the dependencies.
On our end, we do a few more things apart from the version bump which are listed out in this workflow - 
1. updateSHAs
2. spotlessApply
3. Call the changelog helper 

3 is what is responsible for adding in the dependency entries within the CHANGELOG, which it does currently to the `Unreleased 3.0` section. Example: https://github.com/opensearch-project/OpenSearch/pull/7340/files

We then manually backport these version bumps, which means we move the entry within the CHANGELOG to 2.x 

I was proposing to add these entries to `Unreleased 2.x` by default, that way the backport workflow can do its magic and we get rid of the manual backporting process for dependencies with an added bonus that `2.x` will always have the latest versions.